.github/ISSUE_TEMPLATE/c-feature-request.yml

```
@@ -0,0 +1,53 @@
+name: Submit a feature request
+description: Do you have ideas or suggestions for new features? Submit them here!
+title: "[Feature Request]: "
+labels: ["needs-triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to submit a feature request!
+        Please take the time to fill out the information below so that we can better understand your request.
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
```

issue  -> feature request?

---

I'd prefer 'issue' here, as I don't want people confused about where to look. As not all issues are labeled properly, I'd want to encourage them to look through all issues.

---

Maybe rephrase is so both terms are used:
```suggestion
      label: Is there already an issue requesting this feature?
```